projects/jdwp/tests/BUCK

```diff
@@ -4,6 +4,9 @@ python_test(
   name = "tests",
   deps = [
     "//projects/jdwp:lib",
```

Comment:
Could you remove the definition of "lib" target from projects/jdwp/BUCK ? We should not need it anymore. Instead tests should depend on "//projects/jdwp/defs:defs"